stream_alert_cli/terraform_generate.py

```diff
@@ -18,6 +18,7 @@
 import os
 
 from stream_alert_cli.logger import LOGGER_CLI
+from stream_alert.shared import metrics
```

Comment:
Nit: this would imported before `stream_alert_cli.logger`

Reply:
@jacknagz pylint cares about the order of each import group (stdlib then third-party packages), but I don't think it cares about the order within each group